_tests/e2e/package-lock.json

```diff
@@ -1,5717 +0,0 @@
-{
```

Comment:
@RubyGemTSL Sure we want to delete this? My understanding was that we should be comitting `package-lock.json` for wherever we're using Node in a  similar fashion to `Gemfile.lock` wherever we're using Ruby?

Reply:
👍 pushed it back.